fury-on-gke/manifests/monitoring/kustomization.yaml

```
@@ -0,0 +1,21 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../../vendor/katalog/monitoring/alertmanager-operated
+  - ../../vendor/katalog/monitoring/grafana
+  - ../../vendor/katalog/monitoring/kube-proxy-metrics
+  - ../../vendor/katalog/monitoring/kube-state-metrics
+  - ../../vendor/katalog/monitoring/eks-sm
```

```suggestion
  - ../../vendor/katalog/monitoring/gke-sm
```
:warning: remmeber to update the README.md also